src/account/handle.rs

```diff
@@ -138,6 +138,12 @@ impl AccountHandle {
         Ok(all_addresses.to_vec())
     }
 
+    /// Returns all public addresses of the account
+    pub(crate) async fn public_addresses(&self) -> Vec<AccountAddress> {
```

Comment:
This name is a bit inconsistent (doesn't have the `list_`), but tbh I think this `list_` is unnecessary, it just returns a Vec, we don't usually put a `list_` for that kind of functions.

Reply:
Agree, should I remove from all other functions?

Reply:
Yeah, not necessarily in this PR though, as you prefer

Reply:
Opened an issue for it https://github.com/iotaledger/wallet.rs/issues/1441